src/main/java/org/spongepowered/api/world/ChunkTicketManager.java

```diff
@@ -188,6 +188,13 @@
          */
         int getMaxNumChunks();
 
+        /**
+         * Gets the world associated with this ticket during it's creation
+         *
+         * @return The world associated with this ticket
+         */
+        World getWorld();
```

Comment:
SpongeVanilla doesn't have any of the chunk loading wrapper as it's a wrapper for Forge. It shouldn't be used in vanilla at all, right?